tools/tracer/tools/configure/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+cmake_minimum_required( VERSION 2.6.2 FATAL_ERROR )
+project(mfx-tracer-config)
+
+set(headers "strfuncs.h" "config_manager.h")
+set(sources "config_manager.cpp" "main.cpp")
+
+set(no_warnings "-Wno-unknown-pragmas -Wno-unused")
+set(debug_flags "-g -D_DEBUG -O0 -Wall ${no_warnings} -Wformat -Wformat-security")
+set(release_flags "-DNDEBUG -O2 -D_FORTIFY_SOURCE=2 -fstack-protector -Wall ${no_warnings} -Wformat -Wformat-security")
+
+set(CMAKE_C_FLAGS   "-pipe -fPIC -fpie")

```

Comment:
NO. you can _not_ override user provided flags. You are doing exactly that here. You can add, you can't delete.

Reply:
fixed